docs/stdlib/datetime.rst

```diff
@@ -101,35 +101,34 @@ Dates and Times
 
 .. _ref_std_datetime_intro:
 
-EdgeDB has two classes of date/time types:
+EdgeDB has two varieties of date/time constructs:
```

Comment:
This feels like the issue of using "set" outside the context of a proper set. If we say "two types" here, my fear is that a reader will assume EdgeDB has two types that can be used for datetime values. Then, we are showing them four types (`datetime`, `local_datetime`, `local_date`, and `local_time`) spread across two... not sure what to call them. Paradigms?

Is there another word that would work better, or should we say we have four types? I'm reluctant to say we have four types unless we split down to four bullet points... but the point is really just the separation between timezone-aware and local so I'm not sure that makes sense. On the other hand, I'm reluctant to say two "types" because there are four literal types.

I'll try to come up with something that will work, but I'm happy to hear suggestions.

Reply:
What do you think of this to get around the issue I described?

```rest
EdgeDB offers two ways of representing date/time values:
```